docs/api.rst

```diff
@@ -84,4 +84,5 @@ There are two types of tables:
 
 .. autoclass:: Table
     :members:
-    :special-members:
+    :inherited-members:
```

Comment:
```suggestion
    :inherited-members: HeaderBase
```

This would hide the unwanted methods, but `description` we also want to have. Don't know yet how to integrate that.

Reply:
One solution would be to remove `description` from `HeaderBase` and implement in the deriving classes instead, then we could also adjust the docstring accordingly.

Reply:
But there is also `meta`, which we cannot easily remove from `HeaderBase` since we have some custom code there how to handle it. I think it's more elegant to keep both in the base class and find a generic docstring instead.

Reply:
But since this also relates to the other classes I suggest to tackle this issue in a separate PR and stay with the current solution of hiding those for now.

Reply:
I crated https://github.com/audeering/audformat/issues/186 for `description`